tensorflow/compiler/tf2tensorrt/utils/trt_tensor_proxy.h

```diff
@@ -175,7 +173,7 @@ class ITensorProxy {
       case TensorType::kSIMPLE:
         return simple_tensor_->setName(name);
     }
-    assert(0 && "Unsupported itensor_ type");
+    CHECK(0 && "Unsupported itensor_ type");
```

Comment:
LOG(FATAL)  << followed by the message and the value for is used.
Similar comment for all the CHECK(0& below